js/reunion.js

```diff
@@ -0,0 +1,51 @@
+import { getMeeting } from './api/meetings.js';
+
+var url = new URL(window.location);
+
+const searchParams1 = new URLSearchParams(url.search);
+const param = searchParams1.get('meetingId');
+
+const meetInfo = async () => {
+  const meet = await getMeeting(param);
+  buildInfo(meet);
+};
+
+const countryOption = document.querySelector('#country-option');
+const reunionTitle = document.querySelector('#reunion-title');
```

Comment:
Se debería usar nombres en inglés
```suggestion
const meetingTitle = document.querySelector('#reunion-title');
```